alert-database/src/main/resources/scripts/init_alert_db.sql

```diff
@@ -313,3 +313,16 @@ create table if not exists PUBLIC.DATABASECHANGELOG
     LABELS        VARCHAR(255),
     DEPLOYMENT_ID VARCHAR(10)
 );
+
+-- In order for the sequences to match up correctly in hibernate with postgres set the increment to hibernates default allocation size of 50.
+ALTER SEQUENCE alert.audit_entries_id_seq INCREMENT 50;
```

Comment:
Nice. I'm glad you recognized the need for this. It took me forever to figure out how to synchronize with hibernate's generation strategy.